playwright/_impl/_connection.py

```diff
@@ -260,11 +265,9 @@ def _dispatch(self, msg: ParsedMessagePayload) -> None:
                     g.switch(self._replace_guids_with_channels(params))
             else:
                 object._channel.emit(method, self._replace_guids_with_channels(params))
-        except Exception:
-            print(
-                "Error dispatching the event",
-                "".join(traceback.format_exception(*sys.exc_info())),
-            )
+        except BaseException as exc:
```

Comment:
Please re-add how it previously was. With the current was of logging it the stack does not get logged. So the user never know in which of his event listeners an exception got thrown. Consider the following example:

```py
import sys
import traceback
def main():
    def sub():
        def sub_sub():
            raise Exception("abc")
        sub_sub()
    sub()
try:
    main()
except Exception as exc:
    print(f"Event Listerner Error : {exc}")
    print(20 * "-")
    print(
            "Error dispatching the event",
            "".join(traceback.format_exception(*sys.exc_info())),
        )
```

Reply:
Will do tomorrow :)

Reply:
@mxschmitt fixed